shell/app/common/components/erda-icon/erda-icon.tsx

```diff
@@ -101,6 +102,15 @@ const COLOR = {
   white: '#ffffff',
   'light-primary': '#6a549e19', // rgba($primary, .1)
   'shallow-primary': '#6a549e99', // rgba($primary, .6)
+  'shallow-gray': '#bbb',
+  'black-gray': 'rgba(0, 0, 0, 0.85)',
+  'shallow-gray-1': 'rgba(0,0,0,0.4)',
+  'shallo-gray-2': 'rgba(0,0,0,.2)',
+  'shallow-gray-3': 'rgba(0,0,0,.8)',
+  'shallow-gray-5': 'rgba(0,0,0,0.5)',
+  'shallow-gray-6': 'rgba(0,0,0,0.6)',
+  'danger-red': '#df3409',
+  pink: 'rgba(106, 84, 158, 0.8)',
```

Comment:
add a new prop "opacity", the color is determined by the basic color and opacity